src/app/dynamic-forms/fields/rich-text/rich-text.component.ts

```diff
@@ -156,22 +179,51 @@ export class RichTextComponent implements AfterViewInit, OnDestroy {
   ngAfterViewInit(): void {
 
     let content: string | Object;
+    const self = this
 
     try {
       content = ((this.field.value as string).includes(`"type":"doc"`)) ? JSON.parse(this.field.value ?? "{}") : this.field.value;
     }
     catch (error) {
       content = this.field.value;
     }
-
+    
     if (this.canEdit) {
+      // Disable spellcheck in code blocks
+      const CustomCodeBlockLowlight = CodeBlockLowlight.extend({
+        addAttributes() {
+          return {
+            spellcheck: { default: 'false' },
```

Comment:
Please use double quotes for strings or back-ticks (where appropriate) for strings